### DISCUSSION TITLE
p:messages: change aria-label

### DISCUSSION THREAD

**Discussion by hgfrey** [primefaces/community#844] [PrimeFaces] [answered]
How can I change the content of aria-label in the anchor element that is rendered for a p:messages element? 

Just using "close" or "Schließen" in German is not barrier free. It should be "close messages" or "Meldungen schließen" in German.

```
<div class="ui-g contentWithMargin">
  <div id="messages" class="ui-messages ui-widget ui-g-12 ui-md-12 ui-lg-12 ui-no-margin" aria-live="polite">
    <div class="ui-messages-error ui-corner-all">
      <a href="#" class="ui-messages-close" onclick="$(this).parent().slideUp();return false;" aria-label="Schließen">
        <span class="ui-icon ui-icon-close"></span>
      </a>
      <span class="ui-messages-error-icon"></span>
      <ul><li role="alert" aria-atomic="true"><span class="ui-messages-error-summary">Bitte geben Sie einen Infotext an. </span></li></ul>
    </div>
  </div>
</div>
```

**Comment by melloware** [comment, marked as answer]
In PrimeFaces 13 that is a server side message see the docs here: https://primefaces.github.io/primefaces/13_0_0/#/core/localization  So you would just need to update `Messages_de.properties` in your project.

In PrimeFaces 14 that will be handled by a client side JS message in the `locale-de.js` file: https://github.com/primefaces/primefaces/blob/master/primefaces/src/main/resources/META-INF/resources/primefaces/locales/locale-de.js

**Reply by hgfrey** [reply]
fine! that works.
